src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -124,7 +124,7 @@ private static async Task<ExitCode> Run(Configuration config)
             return ExitCode.CliError;
         }
 
-        if (!config.ServerUri.Scheme.StartsWith("http"))
+        if (!config.ServerUri.StartsWith("http"))
```

Comment:
Just curious why this changed?

Reply:
`ServerUri` is no longer of type `System.Uri`, to support values such as `http://+:5050`. This is required for exposing the server port in containers.